packages/file_selector/file_selector_ios/ios/Classes/FLTFileSelectorPlugin.m

```diff
@@ -0,0 +1,80 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "FLTFileSelectorPlugin.h"
+#import "FLTFileSelectorPlugin_Test.h"
+#import "messages.g.h"
+
+@implementation FLTFileSelectorPlugin
+
+#pragma mark - FLTFileSelectorApi
+
+- (void)openFileSelectorWithConfig:(FLTFileSelectorConfig *)config
+                        completion:(void (^)(NSArray<NSString *> * _Nullable,
+                                             FlutterError * _Nullable))completion {
+  if (self.pendingCompletion) {
+    completion(nil, [FlutterError errorWithCode:@"error"
+                                        message:@"There is already a pending file picker request."
+                                        details:nil]);
+    return;
+  }
+
+  UIDocumentPickerViewController *documentPicker = self.documentPickerViewControllerOverride ?:
+  [[UIDocumentPickerViewController alloc] initWithDocumentTypes:config.utis
```

Comment:
You'll need to clang-format all of the ObjC files (or just run the repo tooling `format` command on the whole package).

Reply:
Done